server/auth/facebookAuthInfo.js

```diff
@@ -0,0 +1,9 @@
+var ip  = process.env.IP || 'http://localhost';
+var port = process.env.PORT || 8080;
+module.exports= {
+  facebookAuth: {
+    clientID : process.env.FACEBOOK_CLIENTID,
+    clientSecret: process.env.FACEBOOK_CLIENTSECRET,
+    callbackURL: ip  + '/Server/users/facebook/callback'
```

Comment:
Just double checking, would the callbackURL work if `ip` doesn't start with `http://`? `process.env.ip` would not return `http://` in front of the value.